Documentation/NextSteps/Index.rst

```diff
@@ -20,8 +20,8 @@ The documentation is divided into several sections:
 #. The `Extensions <https://docs.typo3.org/typo3cms/extensions/>`_ section gathers all
    manuals related to core and public extensions.
 
-This is admittedly a huge mass of documentation and you may feel lost
-about where to look next. We recommend the following documents for further studies
+This can be quite overwhelming for people new to the CMS and you may feel lost
+and unsure about where start next. We recommend the following documents for further reading
```

Comment:
I suggest minor change: "about where to start". 

Apart from that, great changes! :thumbsup: 

I will push another commit and merge.